integration-tests/contracts/ethereum_vrf_contracts.go

```diff
@@ -434,6 +435,10 @@ type EthereumVRFv2Consumer struct {
 	consumer *eth_contracts.VRFv2Consumer
 }
 
+func (v *EthereumVRFv2Consumer) ChangeEVMClient(newClient blockchain.EVMClient) {
```

Comment:
@smartcontractkit/test-tooling-team this is done for setting clients when performing requests in parallel. This seems to me as an incorrect approach, but what would you suggest to do?

Reply:
One way around this would be to have a method like `NewEthereumVRFv2Consumer` which will take the new client and use that backend to get a[ new object](https://github.com/smartcontractkit/chainlink/blob/develop/core/gethwrappers/generated/vrf_consumer_v2/vrf_consumer_v2.go#L104) of the contract.